js/functions.js

```diff
@@ -0,0 +1,47 @@
+// Функция для проверки длины строки
+
+function checksStringLength(string, number) {
```

Comment:
Лучше называть обезличенными глаголами 
```suggestion
function checkStringLength(string, number) {
```